### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
## Summary

Pascal broke the Internet (again).
Here is the fix.

@swissspidy Found some misspellings.

## User-facing changes

None.

### CODE REVIEW

**Comment by szepeviktor** [comment]
`.typos.toml` configuration for the willing.

```toml
[files]
extend-exclude = [
    ".git/",
    "includes/data/fonts/fonts.json",
    "packages/fonts/src/fonts.json",
    "tests/phpunit/integration/data/amp.dev.html",
]
ignore-hidden = false

[default]
extend-ignore-re = [
    "Conde Nast",
    "Barceloneta",
    "st, nd, rd",
    "'__WEB_STORIES_ENCODED__\\S+'",
    "Hel\\S{1,2}lo",
    "Hel\\[l<b><i>o",
    "Hell\\[o Worl\\]d",
    "editorStateHel",
]

[default.extend-words]
"optin" = "optin"
"parms" = "parms"
"seeked" = "seeked"
"xdescribe" = "xdescribe"
```

File: packages/e2e-tests/src/specs/editor/shopping/schema.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "type": "string"
     },
     "productUrl": {
-      "description": "Links to the product's website, whne the buy now button is clicked.",
+      "description": "Links to the product's website, when the buy now button is clicked.",
```
Comment (swissspidy) [inline packages/e2e-tests/src/specs/editor/shopping/schema.json:12]: Upstream fix: https://github.com/ampproject/amphtml/pull/39559

File: packages/story-editor/src/components/colorPicker/insertStop.js [hunk 2]
```diff
@@ -40,18 +40,18 @@ function insertStop(stops, newPosition) {
...
   const following = stops[followingIndex];
 
-  // Find the ratio the new stop is from the preceeding stop
-  const diffNewToPreceeding = newPosition - preceeding.position;
-  const diffFollowingToPreceeding = following.position - preceeding.position;
-  const ratio = diffNewToPreceeding / diffFollowingToPreceeding;
+  // Find the ratio the new stop is from the preceding stop
+  const diffNewTopreceding = newPosition - preceding.position;
+  const diffFollowingTopreceding = following.position - preceding.position;
+  const ratio = diffNewTopreceding / diffFollowingTopreceding;
```
Comment (swissspidy) [inline packages/story-editor/src/components/colorPicker/insertStop.js]:
```suggestion
  // Find the ratio the new stop is from the preceding stop
  const diffNewToPreceding = newPosition - preceding.position;
  const diffFollowingToPreceding = following.position - preceding.position;
  const ratio = diffNewToPreceding / diffFollowingToPreceding;
```

**Review by swissspidy — APPROVED** [review]

**Comment by swissspidy** [comment]
@szepeviktor Thank you! Good idea to add this GitHub Action.

**Comment by szepeviktor** [comment]
> Good idea to add this GitHub Action.

Here is your membership to the Library of Viktor Workflows.

https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows

**Pull request merged**